src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const namePartner = person.sex === 'm' && person.isMarried
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {person.name}
+      </h2>
+
+      { person.age && (
+        <p className="Person__age">
+          I am
+          {' '}
+          {person.age}
+        </p>
+      )
+      }
```

Comment:
please check the spaces
```suggestion
      {person.age && (
        <p className="Person__age">
          I am
          {' '}
          {person.age}
        </p>
      )}
```